server/server.js

```diff
@@ -972,11 +974,18 @@ let indexHTML = fs.readFileSync("./dist/index.html").toString();
                     for (let i = 0; i < monitorListData.length; i++) {
                         if ((importHandle == "skip" && monitorNameListString.includes(monitorListData[i].name) == false) || importHandle == "keep" || importHandle == "overwrite") {
 
+                            if (backupData.version.includes("1.7")) {
+                                var retryInterval = monitorListData[i].retryInterval;
+                            } else {
+                                var retryInterval = 0;
+                            }
```

Comment:
```suggestion
                            let retryInterval = 0;

                            if (backupData.version.includes("1.7")) {
                                retryInterval = monitorListData[i].retryInterval;
                            }
```

Reply:
Thank you, idk why i didn't get that in mind.